src/content/docs/en/guides/view-transitions.mdx

```diff
@@ -465,6 +455,13 @@ If you have code that sets up global state, this state will need to take into ac
 
 Module scripts are only ever executed once because the browser keeps track of which modules are already loaded. For these scripts, you do not need to worry about re-execution.
 
+:::tip
+Because module scripts that add event listeners for `DOMContentLoaded` only execute once, you will also need to add listeners for a [Lifecycle event](#lifecycle-events).
+:::
+
+See the [Update Scripts Tutorial](/en/tutorials/add-view-transitions/#update-scripts) for more information.
+
+
```

Comment:
This is an important caveat to communicate and deserves more elaboration. I think `Full site view transitions (SPA mode)` would be the appropriate place to explain the architecture that ultimately makes this necessary.